index.js

```diff
@@ -0,0 +1,45 @@
+let image = document.querySelector(".profile-pic");
```

Comment:
Do not use `let` unless you want to reassign it to another reference.. If its the same reference you can use `const`